web/apps/serve.go

```diff
@@ -69,6 +69,12 @@ func Serve(c echo.Context) error {
 	}
 
 	if file == "" || file == route.Index {
+		if !route.Public {
+			if handled := middlewares.CheckOAuthClientsLimitExceeded(c); handled != nil {
```

Comment:
Are you sure about that? I think `middlewares.CheckOAuthClientsLimitExceeded` returns `nil` when it does the redirection

Reply:
Hum, you're right. This should probably be something like `handled, err := middleware…`